pkg/connectivity/pod-to-pod.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/openservicemesh/osm-health/pkg/common"
 	"github.com/openservicemesh/osm-health/pkg/envoy"
 	"github.com/openservicemesh/osm-health/pkg/kubernetes/namespace"
-	"github.com/openservicemesh/osm-health/pkg/kubernetes/pod"
+	"github.com/openservicemesh/osm-health/pkg/kubernetes/podhelper"
```

Comment:
what prompted this rename? (should we also rename the rest of the packages to namespaceHelper/envoyHelper for parallelism in a separate PR?)

Reply:
nit: lets figure out consistent case naming because now we seem to have container_logs, pod-to-pod and podHelper

Reply:
What prompted the rename is that I previously used a `pod *v1.pod` in the same code context. Golang wasn't able to determine if `pod` here referred to the `pod *v1.pod` variable or the pod helper in `pkg/kubernetes/pod`. That's the same reason why we also have the package name `pkg/kuberneteshelper` to prevent confusion between our own k8s helper pkg and the upstream k8s package. cc @SanyaKochhar

Reply:
Created the following issues to address our naming conventions at a later date.

https://github.com/openservicemesh/osm-health/issues/53
https://github.com/openservicemesh/osm-health/issues/54

Reply:
ooh yes definitely makes sense!

Reply:
Thanks for opening the issues!